pyccel/ast/utilities.py

```diff
@@ -218,7 +218,8 @@ def builtin_import(expr):
     if source == 'pyccel.decorators':
         funcs = [f[0] for f in inspect.getmembers(pyccel_decorators, inspect.isfunction)]
         for target in expr.target:
-            if str(target) not in funcs:
+            search_target = target.name if isinstance(target, AsName) else target
+            if search_target not in funcs:
```

Comment:
Are you sure `str()` is no longer needed?

Reply:
you are right `target` must be casted to `str`